README.md

```diff
@@ -2,7 +2,11 @@
 
 Run [VS Code](https://github.com/Microsoft/vscode) on any machine anywhere and access it in the browser.
 
-![Screenshot](./doc/assets/screenshot.png)
+*****
+SPECIAL ANNOUNCEMENT: We're making it easier to access your code-server instance securely from any device. To gain pre-release access, [join our alpha program](https://codercom.typeform.com/to/U4IKyv0W).
```

Comment:
We should put this down by the install so we aren't too spammy. Maybe like an easiest setup option or somethin